src/components/Footer/index.js

```diff
@@ -117,127 +117,129 @@ const Page = ({ children, ...props }) => (
   </Link>
 )
 
-const Footer = () => {
+const UpperFooter = () => (
```

Comment:
`<UpperFooter />` and `<LowerFooter />` are quite large. Please put them into new files.

Reply:
done.